support: don't try to run compiled code

When cross-compiling, we can't run the compiled code.

Consider the compiler returning success as an indication
it is working.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN tvheadend.orig/support/configure.inc tvheadend-c7d0335eb10d02b780423bef8e7c740e422fff13/support/configure.inc
--- tvheadend.orig/support/configure.inc	2013-12-01 01:24:26.000000000 +0100
+++ tvheadend-c7d0335eb10d02b780423bef8e7c740e422fff13/support/configure.inc	2013-12-03 22:34:57.547762178 +0100
@@ -265,8 +265,6 @@
 EOF
   $CC $CFLAGS $LDFLAGS $TMPDIR/$$.c -o $TMPDIR/$$.bin $opt #&> /dev/null
   RET=$?
-  [ $RET -eq 0 ] && $TMPDIR/$$.bin
-  RET=$?
   rm -f $TMPDIR/$$.{c,bin}
   return $RET
 }
